### PR TITLE
Add tool to anonymize HAR files

### LINKED ISSUE #24: Add a HAR Anonymizer
Make it easy to anonymize HARs by replacing UUIDs and randomizing things like date. This will be helpful both for when we want to add more integration tests and if ever we'll want people to submit their HARs for bugfixing.

I believe the HAR download and the anonymization could be done right out of the extension.

### PR DESCRIPTION
**Pull request by akirk**
As mentioned in our [Working with the Command-line](https://github.com/pento/free-as-in-speech/blob/anonymize-har/docs/contributors/command-line.md) docs, HAR files can contain PII (Personally Identifiable Information) which we want to avoid.

Thus, this implements a tool that can rewrite HAR files to be more anonymous. Since it contains a list of rules, there might still be some rules missing so that it needs to be reviewed from time to time.

## Testing instructions

Run the script on a HAR file:
```
$ node packages/fetch-from-har/bin/anonymize-har.js test.har
Reading test.har...

The following number of items were replaced:
 - 762 timestamp
 - 13 ipv4
 - 5 ipv6
 - 78 email
 - 948 uuid
 - 1 wixSite
 - 1 _wixUIDX
 - 1 mediaToken
 - 1 wixSession2
 - 10 authorization
 - 3 session_id
 - 176 request_id
Wrote anonymized-test.har.
```

Then verify that the anonymized file contains changes and that it still works fine:
```
$ diff test.har anonymized-test.har | wc -l
   21608
$ diff test.har anonymized-test.har | $EDITOR
$ open anonymized-test.har|
```

Fixes #24

### CODE REVIEW

File: packages/fetch-from-har/bin/anonymize-har.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+		replacement: randomBase62String,
+	},
+	session_id: {
+		regex: valueRegex( 'session_id', 'g' ),
+		replacement: randomBase62String,
+	},
+	request_id: {
+		regex: valueRegex( 'x-wix-request-id', 'g' ),
+		replacement: randomBase62String,
+	},
```
Comment (yoavf) [inline packages/fetch-from-har/bin/anonymize-har.js:167]: Should we also anonymize the user-agent?
```
 {
              "name": "User-Agent",
              "value": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.16; rv:86.0) Gecko/20100101 Firefox/86.0"
            },
```

File: packages/fetch-from-har/bin/anonymize-har.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+};
```
Comment (yoavf) [inline packages/fetch-from-har/bin/anonymize-har.js:172]: There are a few other cookies (and matching headers) that we might want to anonymize or even completely remove (assuming extracting still works):

- `svSession` - "Used in connection with user login", and `XSRF-TOKEN` -"Used for security reasons" . `TS*`  - "Used for security and anti-fraud reasons" [Reference here](https://support.wix.com/en/article/cookies-and-your-wix-site) 
- `_wixAB3` - probably some ab testing
- `_ga` - Google Analytics
- `_gcl_au` - Google Tag Manager

For the headers, can we anonymize/remove:
- x-seen-by (wix specific)

**Review by yoavf — COMMENTED** [review]
Tested, works well - both on anonymizing the file, and making sure the file still works with `export-from-har.js` 

Added a few suggestions for additional data to anonymize.

**Review by pento — CHANGES_REQUESTED** [review]
We'll probably want to reuse this code when we allow users to submit debug info, but it looks relatively straightforward to convert the code to something importable.

One minor thing to change, there shouldn't be a `package-lock.json` in the package folder: this can be fixed by removing that file, and running `npm i` in the base folder.

**Comment by akirk** [comment]
@pento done and rebased.

**Review by pento — APPROVED** [review]
🚢

**Pull request merged**